helloworld-impl/src/main/java/sample/helloworld/impl/HelloServiceImpl.java

```diff
@@ -37,32 +33,22 @@
   }
 
   @Override public ServiceCall<NotUsed, String> hello(String id) {
-    System.out.println("hello");
     return request -> {
-      // Look up the hello world entity for the given ID.
       PersistentEntityRef<HelloCommand> ref = persistentEntityRegistry.refFor(HelloWorld.class, id);
-      // Ask the entity the Hello command.
-      greetingsTopic();
       return ref.ask(new Hello(id, Optional.empty()));
```

Comment:
@ignasi35 Thanks for the PR. It was really helpful but I still am not able to consume any message.
I was invoking `greetingTopic()` manually because I am unsure that it's invoked every time and message is persisted.